keyboards/ergodox_ez/keymaps/rmw/keymap-mac.c

```diff
@@ -0,0 +1,236 @@
+    _______, KC_PERC, KC_CIRC, KC_LBRACKET, KC_RBRACKET, _______, _______, 
+    _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, KC_QUES, KC_SLSH, KC_COLON, _______, 
+    _______, _______, _______, _______, _______, _______, LCTL(KC_LEFT), 
+    _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, LGUI(KC_DOWN), _______), 
+
+  [MEDIA] = LAYOUT_ergodox(_______, _______, KC_MS_BTN4, _______, KC_MS_BTN5, _______, _______, 
+    _______, KC_MS_WH_UP, KC_MS_WH_LEFT, KC_MS_UP, KC_MS_WH_RIGHT, _______, _______, 
+    _______, KC_MS_WH_DOWN, KC_MS_LEFT, KC_MS_DOWN, KC_MS_RIGHT, _______, 
+    _______, _______, KC_MS_ACCEL0, KC_MS_ACCEL1, KC_MS_ACCEL2, _______, _______, 
+    _______, _______, _______, _______, KC_MS_BTN3, 
+    _______, _______, _______, KC_MS_BTN1, KC_MS_BTN2, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, 
+    _______, _______, KC_AUDIO_VOL_DOWN, KC_AUDIO_MUTE, KC_AUDIO_VOL_UP, _______, _______, 
+    KC_MEDIA_REWIND, KC_MEDIA_FAST_FORWARD, _______, _______, _______, 
+    _______, _______, _______, _______, KC_MS_BTN1, KC_MEDIA_PLAY_PAUSE), 
+
+  [ADJUST] = LAYOUT_ergodox(_______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, TO(EDIT), TO(EXCEL), TO(FSYM), TO(JSYM), TO(MEDIA), 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, TO(QWERTY), DF(MINIMAK4), DF(QWERTY), _______, 
+    _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______, _______), 
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case VRSN:
+      if (record->event.pressed) {
+        SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+      }
+      return false;
+    case FORM_GET:
+      if (record->event.pressed) {
+        tap_code(KC_BTN1);
+        tap_code16(G(KC_A));
+        tap_code16(G(KC_C));
+      }
+      return false;
+    case FORM_PUT:
+      if (record->event.pressed) {
+        tap_code(KC_BTN1);
+        tap_code16(G(KC_A));
+        tap_code16(G(KC_V));
+      }
+      return false;
+  }
+  return true;
+}
+
+void matrix_scan_user(void) {
```

Comment:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

Reply:
I tried this somewhere upthread. Unfortunately, it doesn't work: It seems that LT holds don't result in LED changes until after release, then they're showing that you're on e.g. EDIT when you're back on QWERTY. Is there some option I'm missing that fixes that?

Reply:
Odd... Could you try the [`rgb_layer keymap`](https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/keymaps/rgb_layer/keymap.c#L226-L270)?

Reply:
NVM, apparently just unplugging and replugging the board after the programming reboot fixed it? It was seriously messed up after boot until that.